srtcore/core.cpp

```diff
@@ -8439,6 +8439,7 @@ int CUDT::processData(CUnit *in_unit)
         }
 
         bool excessive = true; // stays true unless it was successfully added
+      int avail_bufsize = -1;
```

Comment:
`avail_bufsize` is declared and set later on. This line looks like a merge error.

Reply:
I'd agree about the format, but this is intended due to being later used in the log.